common/src/main/webapp/sass/explore.scss

```diff
@@ -37,8 +37,6 @@ $tree-section-width: 300px;
   }
 
   .p-button.p-button-sm {
-    max-width: 4rem;
```

Comment:
@cquiroz Was there a reason you set the max-width for the buttons in the summary table toolbar to 4rem? I removed it so the whole text of `+ Sidereal` would display.

Reply:
I don't recall